README.md

```diff
@@ -34,6 +34,15 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - ms/push-image-prod:
+          context: microservices-okta
+          requires:
+            - ms/build-image
+          filters:
+            tags:
+              only: /^v.*/
+          branches:
+            ignore: /.*/
```

Comment:
It is important that `branches` filter ignore all commits. This way only tagged builds would publish to production AWS ECR and avoid polluting it with each commit / PR image.